index.d.ts

```diff
@@ -66,6 +66,10 @@ declare module 'plaid' {
     };
   }
 
+  interface AccountWithIdentity extends Account {
```

Comment:
nit: can we call this `AccountWithOwners` ?

Reply:
+1 to be consistent with the API